docs/explanations/faq.rst

```diff
@@ -1,23 +1,68 @@
+Troubleshooting and FAQ
+=======================
+
+Please visit the `IDAES discussions board on Github <https://github.com/IDAES/idaes-pse/discussions>`_ for more questions and answers.
+
+.. contents::
+    :depth: 2
+    :local:
+
 Frequently Asked Questions
-==========================
+--------------------------
+
+Help! My model does not solve!
+``````````````````````````````
+
+The IDAES team does not have a lot of resources for supporting users to solve their models, so before contacting the IDAES team for help, make sure you have tried all the steps below.
+
+  1. First, check the output logs for any warnings or cautions as these may indicate that there are problems with your model or code that need to be addressed before you try to solve it. Also make sure that the code is making it to the solver call and that the issue does not lie with the model construction.
+  2. Second, check the final solver status to understand how the solver is failing as this can give you hints as to what is wrong. If the solver status is infeasible, it generally indicates that the model is actually infeasible and that you need to fix something in your model. It is also helpful to look at the solver output (`solver.solve(model, tee=True)`) as there is often useful information in the solver traces.
+  3. Next, try using the :ref:`IDAES Diagnostics workflow and toolbox <explanations/model_diagnostics/index:Model Diagnostics Workflow>` and ensure there are no structural or numerical issues in your model.
+  4. If the model still fails to solve, try using the Degeneracy Hunter or SVD toolboxes to look for more complex issues.
```

Comment:
Are there links for either Degeneracy Hunter or the SVD toolbox?  Even if it is just a link pointing to the module in the code browser, that would help.